tensilelite/Tensile/KernelWriter.py

```diff
@@ -995,7 +995,7 @@ def checkLocalReadFIFOFull(currentMFMA, fifo, lrItems, numLR, numLREven):
 
       oneBufferScheduling = kernel["1LDSBuffer"] or kernel["DirectToLdsA"] or kernel["DirectToLdsB"]
       
-      def hasDependency(lr: DSLoadInstruction, inst: MFMAInstruction | SMFMAInstruction | Instruction) -> bool:
+      def hasDependency(lr: DSLoadInstruction, inst: Instruction) -> bool:
```

Comment:
You could support Python 3.6 via the following type hint
```
Union[MFMAInstruction, SMFMAInstruction, Instruction]
```

This can be imported from the `typing` module

Reply:
Ya, I know, this was just a quick fix, maybe amend this by #1346.